sdk/core/core-xml/src/xml.ts

```diff
@@ -99,14 +100,17 @@ export async function parseXML(str: string, opts: XmlOptions = {}): Promise<any>
 
   // Remove the <?xml version="..." ?> node.
   // This is a change in behavior on fxp v4. Issue #424
-  if (parsedXml["?xml"]) {
+  if (objectHasProperty(parsedXml, "?xml")) {
```

Comment:
Looks good but this seems like a big hammer for this fix, could we just instantiate the type variable in the method call in L99 with `Record<string, any>`?